rust-toolchain.toml

```diff
@@ -2,6 +2,6 @@
 # This specifies the version of Rust we use to build.
 # Individual crates in the workspace may support a lower version, as indicated by `rust-version` field in each crate's `Cargo.toml`.
 # The version specified below, should be at least as high as the maximum `rust-version` within the workspace.
-channel = "1.58.1"
```

Comment:
Curious, was this intentional? :)

Reply:
Oh yeah, that one.^^ I guess it was semi-intentional. Xshell wouldn't work with 1.58.1 so I had to lift the requirement. But I wanted to ask you explicitly about this since I didn't know what the best approach was. That, I completely forgot.

Reply:
Heh, we both missed that. Yeah, it's better to land compiler upgrades in a separate PRs, as things might get wrong